src/com/goide/ui/GoConfigurable.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2013-2015 Sergey Ignatov, Alexander Zolotov, Mihai Toader, Florin Patan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.goide.ui;
+
+import com.goide.GoIcons;
+import com.goide.settings.GoApplicationSettings;
+import com.intellij.openapi.options.ConfigurationException;
+import com.intellij.openapi.options.SearchableConfigurable;
+import com.intellij.openapi.project.Project;
+import org.jetbrains.annotations.Nls;
+import org.jetbrains.annotations.NotNull;
+
+import javax.swing.*;
+
+public class GoConfigurable implements SearchableConfigurable {
+
+  private GoConfigurableForm configurableForm;
```

Comment:
It's more compact. Extracting some forms to separate class makes sense when you're going to reuse this form or if the form is quite complicated and have some non-trivial logic in initialization/disposing parts.